### PR TITLE
rgw: add metadata and data sync related cmd into radosgw-admin usage

### PR DESCRIPTION
**Pull request by Leeshine**
data sync relaetd cmd :
+ radosgw-admin data sync status
+ radosgw-admin data sync init
+ radosgw-admin data sync run

metadata sync related cmd:
+ radosgw-admin metadata sync status
+ radosgw-admin metadata sync init
+ radosgw-admin metadata sync run

Signed-off-by: lvshanchun <lvshanchun@gmail.com>

### CODE REVIEW

**Comment by cbodley** [comment]
thanks @Leeshine! could you please add the commands for `metadata sync status/init/run` while you're at it?

**Comment by Leeshine** [comment]
ok, I will add it later.  @cbodley

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -143,6 +143,12 @@ void usage()
   cout << "  zone placement add         add a zone placement target\n";
   cout << "  zone placement modify      modify a zone placement target\n";
   cout << "  zone placement rm          remove a zone placement target\n";
+  cout << "  data sync status           get data sync status of the specified source zone\n";
+  cout << "  data sync init             init data sync for the specified source zone\n";
+  cout << "  data sync run              run data sync for the specified source zone\n";
+  cout << "  metadata sync status       get metadata sync status\n";
+  cout << "  metadata sync init         init metadata sync\n";
+  cout << "  metadata sync run		run metadata sync\n";
```
Comment (cbodley) [inline src/rgw/rgw_admin.cc]: it looks like there's a tab in that last line that won't match what's in `help.t`

Comment (Leeshine) [reply src/rgw/rgw_admin.cc]: I have update it @cbodley

**Pull request merged**